docs/conf.py

```diff
@@ -18,10 +18,10 @@
 # relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
 #
-import os
-import sys
-
-sys.path.insert(0, os.path.abspath(".."))
+# import os
+# import sys
+#
+# sys.path.insert(0, os.path.abspath(".."))
```

Comment:
readthedocs doesn't install packages in editable mode now, changing the `sys.path` will make Python fail to import `ncollpyde.ncollpyde` module(in Rust).